### PR TITLE
Add `adjust` param to `stat_density2d()` (fix #2860)

### LINKED ISSUE #2860: change bandwidth approach in stat_density2d to match stat_density
`stat_density` currently has the `adjust` parameter which defaults to 1 and scales whatever the default bandwidth would be. `stat_density2d` uses `h`, a length-2 vector which is directly passed to the underlying function in the units of the data.

### PR DESCRIPTION
**Pull request by haleyjeppson**
This PR fixes #2860

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -6,6 +6,8 @@ ggplot2 1.0.1
 * Improvements to order of colours and legend display for continuous colour
   scales extracted from colorbrewer palettes by `scale_*_distiller()` (@jiho, 1076)
 
+* `stat_density2d()` can now take an `adjust` parameter to scale the default bandwidth. (#2860, @haleyjeppson)
```
Comment (hadley) [inline NEWS]: This needs to be in `NEWS.md`

File: R/stat-density-2d.r [hunk 2]
```diff
@@ -5,6 +5,10 @@
 #' @param n number of grid points in each direction
 #' @param h Bandwidth (vector of length two). If `NULL`, estimated
 #'   using [MASS::bandwidth.nrd()].
+#' @param adjust A multiplicate bandwidth adjustment to be used if 'h' is
```
Comment (hadley) [inline R/stat-density-2d.r]:
```suggestion
#' @param adjust A multiplicative bandwidth adjustment to be used if 'h' is
```

File: NEWS (hunk 1 above)
Comment (haleyjeppson) [reply NEWS]: :woman_facepalming: whoops! fixed now.

File: NEWS.md [hunk 3]
```diff
@@ -27,6 +27,8 @@
   more informative warnings when supplied with set aesthetics
   (i.e., `slope`, `intercept`, `yintercept`, and/or `xintercept`)
   and mapped aesthetics (i.e., `data` and/or `mapping`).
+  
+* `stat_density2d()` can now take an `adjust` parameter to scale the default bandwidth. (#2860, @haleyjeppson)
```
Comment (hadley) [inline NEWS.md]: For future reference, the best place to put new bullets is at the top.

**Comment by hadley** [comment]
Thanks @haleyjeppson!

File: NEWS.md (hunk 3 above)
Comment (haleyjeppson) [reply NEWS.md]: Ahh okay, noted!

**Pull request merged**